WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1343,6 +1343,18 @@ decisions = {
 				factor = 0
 				religion = marcionism
 			}
+			modifier = {
+				factor = 10
+				trait = kind
+			}
```

Comment:
Yes, but you are the one who sometimes interprets it differently or applies revisionism. Specifically, you and I are at odds on historical Jew stuff, and there's some historical revisionism on your part there.